aws/table_aws_spot_price_history.go

```diff
@@ -0,0 +1,151 @@
+package aws
+
+import (
+	"context"
+	"errors"
+	"github.com/aws/aws-sdk-go-v2/service/ec2"
+	"github.com/aws/aws-sdk-go-v2/service/ec2/types"
+	"github.com/turbot/steampipe-plugin-sdk/v4/grpc/proto"
+	"github.com/turbot/steampipe-plugin-sdk/v4/plugin"
+	"github.com/turbot/steampipe-plugin-sdk/v4/plugin/transform"
+	"time"
+)
+
+//// TABLE DEFINITION
+
+func tableAwsSpotPriceHistory(_ context.Context) *plugin.Table {
+	return &plugin.Table{
+		Name:        "aws_spot_price_history",
+		Description: "AWS Spot Price History",
+		List: &plugin.ListConfig{
+			Hydrate: listSpotPriceHistory,
+			KeyColumns: []*plugin.KeyColumn{
```

Comment:
Does the API also support filtering based on spot price? It looks like it does, though there are some limitations (from https://docs.aws.amazon.com/cli/latest/reference/ec2/describe-spot-price-history.html#options):
> The Spot price. The value must match exactly (or use wildcards; greater than or less than comparison is not supported).

Reply:
Yes ! It's why I removed this field. 
IMO, i don't see any usecase when an user should search a spot price by price (with or without wildcard).

I can add it if you want to respect the API reference.

Reply:
@aminvielledebatAtBedrock I'm not as familiar with how people would use this API/table, but could someone use this filter to search for instance types/regions/etc., that match a specific spot price, or a spot price using wildcards (as these are supported, though not greater than or less than comparisons unfortunately)?

Reply:
@aminvielledebatAtBedrock One solution would be to have only `=` operator qual support for `spot_price`, so if someone used greater than or less than operators, no `filters` would be passed in and only Postgres filtering would be used, but if someone was looking for a specific spot price (with or without wildcards), then `filters` would be used to help narrow down the results.

However, I think based off of our discussion in https://github.com/turbot/steampipe-plugin-aws/pull/1378#discussion_r1021003008, it's ok to not have key column support for `spot_price` for now.